### PR TITLE
Trace errors in AppInsights

### PR DESCRIPTION
**Pull request by skalitin**
image

### CODE REVIEW

**Comment by denis1stomin** [comment]
Cool! but I have to check it works in my Chromium browser

File: webui/src/store/modules/okr.js [hunk 1]
```diff
@@ -71,11 +77,15 @@ export default {
 
             objectives[objectives.indexOf(obj)].statement = payload.statement;
             state.objectives = objectives;
+
+            telemetry.trackError();
```
Comment (denis1stomin) [inline webui/src/store/modules/okr.js]: not sure it's a failure function

Comment (denis1stomin) [reply webui/src/store/modules/okr.js]: I mean EDIT_OBJECTIVE

**Review by denis1stomin — APPROVED** [review]
Works good but I think should provide functiona names ourself.
Currently in customDimensions I see `method | http://localhost:8080/main.a853ee6e.js:79:6001` which is not something useful for us. Especially to calculate errors statistic.
Let's return back textual functions names.

**Comment by skalitin** [comment]
> Let's return back textual functions names.
Fixed!

**Pull request merged**